docs/standard/class-libraries-multi-targeting.md

```diff
@@ -0,0 +1,283 @@
+---
+title: .NET Class Library Multi-Targeting
+description: Learn how to create .NET class libraries for multiple .NET platforms that can access and wrap platform-specific functionality.
+keywords: .NET, .NET Core
+author: immol
+ms.author: mairaw
+ms.date: 03/19/2018
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+ms.assetid: 375da522-3520-4735-b1de-68c7b71719d5
+ms.workload: 
+  - "dotnet"
+  - "dotnetcore"
+---
+
+# Introduction
+
+Even though .NET Standard 2.0 is very rich in terms APIs, there are rare cases
+where you need to access framework specific APIs (for instance, you might want
+to pop UI for starting an OAuth flow). There are also cases where you want to
+introduce a standard way to access framework-specific functionality (for
+example, using the geolocation APIs).
+
+You can do this via [multi-targeting](frameworks.md) which builds your project
+for multiple framework from a single project. This enables you to use
+conditional compilation with `#if` to call framework specific APIs.
+
+Please note that this topic is primarily about libraries. For application
+development we generally recommend to use multiple projects (one per
+application) and sharing code via Shared Projects.
+
+# Multi-targeting
+
+.NET Standard projects start as targeting a single framework, namely a specific
+version of .NET Standard:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+</Project>
+```
+
+In order to target multiple frameworks, you need to change the name of the
+property from `TargetFramework` to the plural form `TargetFrameworks`. Then you
+can add all the desired frameworks as a semicolon separated list:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461;netcoreapp2.0</TargetFrameworks>
```

Comment:
Do we have a list of TFs somewhere?

Reply:
Yes, it's linked two paragraphs down: [https://docs.microsoft.com/en-us/dotnet/standard/frameworks#supported-target-framework-versions](https://docs.microsoft.com/en-us/dotnet/standard/frameworks#supported-target-framework-versions)

---
In reply to: [175868662](https://github.com/dotnet/docs/pull/4709#discussion_r175868662) [](ancestors = 175868662)

Reply:
Can we add Xamarin stuff on there? iOS, Android, and Mac? Also, Tizen is supported too.